src/briefcase/integrations/java.py

```diff
@@ -70,7 +79,7 @@ def verify(cls, tools: ToolCache, install=True):
         install_message = None
 
         if tools.host_arch == "arm64" and tools.host_os == "Darwin":
-            # Java 8 is not available for macOS on ARM64, so we will require Rosetta.
+            # Java is not available for macOS on ARM64, so we will require Rosetta.
```

Comment:
This isn't true any more; we can get a native ARM Java.

Reply:
That's what I was thinking....so, all the logic to handle rosetta can come out?

Reply:
I think so - this was the only thing we were using it for, AFAIR.